modules/ztp-telco-core-software-versions.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
```

Comment:
Since this topic is identical for both RDS docs, we should consolidate before final merge. cc @imiller0 WDYT